flexget/plugins/input/npo_watchlist.py

```diff
@@ -267,6 +270,8 @@ def _parse_episodes_std(self, task, config, series_name, page, series_info):
             e['npo_name'] = series_info['npo_name']
             e['npo_description'] = series_info['npo_description']
             e['npo_language'] = series_info['npo_language']
+            if subtitle[1]:  # if runtime is available in the subtext
```

Comment:
Hmm, I checked the current way, it and this should actually work already correctly... and the split() always returns a list; (even if the splitted symbol is not there);
and the `if`-statement correctly handles the situation if the key exists in the list. It does not spawn any error with any input.